src/akamai/js/akamaiHD_flash.js

```diff
@@ -242,6 +243,16 @@ require("../../../html5-common/js/utils/constants.js");
      * @param {boolean} rewind True if the stream should be set to time 0
      */
     this.load = function(rewind) {
+      if (loaded && !rewind) return;
+      if (!!rewind) {
+        try {
+          this.callToFlash("load("+rewind+")");
+          loaded = true;
+        } catch (ex) {
+          // error because currentTime does not exist because stream hasn't been retrieved yet
+          OO.log("[Akamai HD]: Load URL", "Failed to rewind video, probably ok; continuing");
+        }
+      }
```

Comment:
Hm, main video plugin does load the stream even if rewind is false - the only thing rewind means is that you should set the currentTime to 0